src/schema/show.ts

```diff
@@ -386,7 +413,38 @@ export const ShowType = new GraphQLObjectType({
       description: "The exhibition title",
       resolve: ({ name }) => (isExisty(name) ? name.trim() : name),
     },
+    nearbyShows: {
+      description: "Shows that are near (~75km) from this show",
+      type: showConnection,
+      args: pageable({
+        size: {
```

Comment:
Why `size`? As in, a connection and `pageable` already gives you `first` and `last` (for specifying number of items to return).